cogs/secretSanta.py

```diff
@@ -0,0 +1,145 @@
+import nextcord
+from nextcord.ext import tasks, commands
+import time
+from datetime import *

```

Comment:
Since you only use the `today` command from datetime, can you do import only `today` instead?
```suggestion
from datetime import today
```